config/values/bursaries.yml

```diff
@@ -0,0 +1,23 @@
+bursaries:
+  skeweekly: £175
+  veteran:
+    total: £40,000
+    instalment: £20,000
```

Comment:
I wonder if this is better as `annual` rather than instalment @MylesJarvis ?

It is 20k per year in equal monthly instalments according to https://getintoteaching.education.gov.uk/funding-and-support/if-youre-a-veteran

So at the moment we are using the term `instalment` for annual amount (variable name) and monthly amount (in content) which might be a bit confusing?

Reply:
Done!